### ISSUE TITLE
Any update?

### ISSUE THREAD

**Issue by aliadibelli** [jagjot2008/EeazyCRM#1]
I am new at python.I downloaded your program.
You did a good jobs. Congratulations.
Will you plan to write and update EeazyCRM?

Have a nice day.

**Comment by jagjot2008** [comment]
Hi! Thanks. I had actually abandoned it because nobody seemed to be interested. I'll think about it.

**Comment by aliadibelli** [comment]
Did you prepare a database model for Activities section?

**Comment by jagjot2008** [comment]
No. I lost interest in the project because nobody was keen. I'm not sure if I'll continue.

**Comment by aliadibelli** [comment]
I want to try adding Activities section. But I am new at pyhon.
Can you help me if I need your help?

**Comment by OmarUsman** [comment]
going to be using this for a project, thanks for making this.

**Comment by vanzhiganov** [comment]
@jagjot2008 currently I was install EeazyCRM in my startup as main CRM for researching and develop API resources for integrating. In quick view, I see that EeazyCRM is a better open source CRM system that many others. Thank you for this.

**Comment by MADHUMITHASIVAKUMARR** [comment]
To get updates on a specific feature or the overall status of the EeazyCRM project, you can:

1. **Check the GitHub Repository**:
   - Look at the **Issues** and **Pull Requests** sections for any recent activity or discussions about features.
   - Review the **Commits** tab to see the latest changes made to the codebase.

2. **Release Notes**:
   - If the repository has a **Releases** section, you can check there for any notes on new features, bug fixes, or updates.

3. **Documentation**:
   - The README or any accompanying documentation may also provide insights into current features or planned updates.

4. **Discussions**:
   - Some repositories have a Discussions section where maintainers and contributors share ideas about features or improvements.

5. **Contact Maintainers**:
   - If you have specific questions about features or updates, consider opening an issue or reaching out directly to the project maintainers.
